tox.ini

```diff
@@ -41,3 +41,11 @@ commands =
   coverage run --branch --include='git_machete/*' --omit='git_machete/tests/*' setup.py test
   coverage report
   coverage html
+
+[testenv:docs]
+description = "Build documentation"
+deps =
+  sphinx==4.1.2
+  sphinx-rtd-theme==0.5.2
+commands =
+  sphinx-build -b html docs/source docs/html
```

Comment:
How do you upload the docs now? can this be automated in the CI on master builds?

Reply:
That's actually interesting, but read_the_docs.io don't use html - that's why I haven't uploaded it. They build the documentation themselves. 
In terms of building on master I would say that we should build 2 documentation, stable (master) and latest (develop) and it is suggested by readthedocs.io (there might be a need to add some tags to git, but we will see).
Automating it seems to be easy, but I think you have to either add specific hooks or synchronize github account with readthedocs account and then repos which you have under control you will be able to add to rtd and build their documentation with every push or something like that.

Reply:
Documentation that I pasted on top is hosted on my account and is using docs/read_the_docs branch and my account is not synchronized.

Reply:
Please `resolve conversation` if response is sufficient.

Reply:
Okay! lemme leave the conversation open since I suppose this needs to be decided upon (whether to just add webhooks to GH or add more automation) before this PR is merged

Reply:
I'm afraid that it will be only known for sure after one of the maintainers will sync their account with readthedocs and try it out. I remember that some of the permission required were hooks and I suspect that reathedocs is doing it for us, but again I would recommend to just try it out.

Reply:
Okay, pls create a separate task for that then! although I hope it's gonna be quick setup (?) once RTD is granted permissions by @git-machete-bot

Reply:
It took me about 5-10min to run it on my branch from my private account so I suspect it's rather easy, but there can be some decision to make like how often update, which version should be available etc. which could delay deliver of this PR.

Issue: https://github.com/VirtusLab/git-machete/issues/210